sci-libs/pdal/pdal-2.4.0.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="A C++ library for translating and manipulating point cloud data"
+HOMEPAGE="https://pdal.io/"
+SRC_URI="https://github.com/PDAL/PDAL/releases/download/${PV}/PDAL-${PV}-src.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/13"
+KEYWORDS="~amd64 ~x86"
+IUSE="postgres test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+DEPEND="
+	dev-libs/jsoncpp
+	net-misc/curl
+	sci-libs/gdal
+	sci-libs/libgeotiff
+	sci-geosciences/laszip
+	sys-libs/libunwind
+	sys-libs/zlib
+	postgres? ( dev-db/postgresql:*[xml] )
+	test? ( sci-libs/gdal[geos,jpeg,png] )
+"
+RDEPEND="${DEPEND}"
```

Comment:
The `test? ( ...)` dep doesn't belong in RDEPEND. Consider swapping DEPEND & RDEPEND to solve this, and adding the test dep in DEPEND.

Also, use := on any deps we link against that expose a subslot.

Reply:
thank you for this feedback.